### PR TITLE
Add is_supported function to the custom lb test

### PR DESCRIPTION
**Pull request by temawi**
@sergiitk

### CODE REVIEW

File: tools/run_tests/xds_k8s_test_driver/tests/custom_lb_test.py [hunk 1]
```diff
@@ -32,6 +32,12 @@
 
 class CustomLbTest(xds_k8s_testcase.RegularXdsKubernetesTestCase):
 
+    @staticmethod
+    def is_supported(config: skips.TestConfig) -> bool:
+        if config.client_lang in _Lang.JAVA:
```
Comment (sergiitk) [inline tools/run_tests/xds_k8s_test_driver/tests/custom_lb_test.py:37]: `_Lang` is an alias: `_Lang = skips.Lang`.

You can either add the alias, or just do `skips.Lang.JAVA`.

File: tools/run_tests/xds_k8s_test_driver/tests/custom_lb_test.py [hunk 2]
```diff
@@ -32,6 +32,12 @@
...
+            return not config.version_lt('v1.47.x')
```
Comment (sergiitk) [inline tools/run_tests/xds_k8s_test_driver/tests/custom_lb_test.py:38]: This method has been removed in https://github.com/grpc/grpc/pull/30124, use `version_gte`

File: tools/run_tests/xds_k8s_test_driver/tests/custom_lb_test.py [hunk 3]
```diff
@@ -32,6 +32,12 @@
 
 class CustomLbTest(xds_k8s_testcase.RegularXdsKubernetesTestCase):
 
+    @staticmethod
+    def is_supported(config: skips.TestConfig) -> bool:
```
Comment (sergiitk) [inline tools/run_tests/xds_k8s_test_driver/tests/custom_lb_test.py:36]: skips must be imported.

**Review by sergiitk — CHANGES_REQUESTED** [review]

File: tools/run_tests/xds_k8s_test_driver/tests/custom_lb_test.py [hunk 4]
```diff
@@ -19,19 +19,27 @@
...
+_Lang = skips.Lang
 
 _EXPECTED_STATUS = grpc.StatusCode.DATA_LOSS
 
 
 class CustomLbTest(xds_k8s_testcase.RegularXdsKubernetesTestCase):
 
+    @staticmethod
+    def is_supported(config: skips.TestConfig) -> bool:
+        if config.client_lang in _Lang.JAVA:
```
Comment (sergiitk) [inline tools/run_tests/xds_k8s_test_driver/tests/custom_lb_test.py:39]: Minor: let's add a quick comment explaining why it's only enabled in Java `v1.47.x`.
Usually I link https://github.com/grpc/grpc/blob/master/doc/grpc_xds_features.md, but I couldn't find your feature there. Should it be added?

**Review by sergiitk — APPROVED** [review]
Please give it a run (locally) with the `--testing_version=v1.41.x` arg to confirm it skips.

File: tools/run_tests/xds_k8s_test_driver/tests/custom_lb_test.py (hunk 4 above)
Comment (temawi) [reply tools/run_tests/xds_k8s_test_driver/tests/custom_lb_test.py:41]: I added a comment, but I didn't mention this doc, will find out if it should get updated.

File: tools/run_tests/xds_k8s_test_driver/tests/custom_lb_test.py [hunk 5]
```diff
@@ -19,19 +19,29 @@
...
 _XdsTestServer = xds_k8s_testcase.XdsTestServer
 _XdsTestClient = xds_k8s_testcase.XdsTestClient
+_Lang = skips.Lang
 
 _EXPECTED_STATUS = grpc.StatusCode.DATA_LOSS
 
 
 class CustomLbTest(xds_k8s_testcase.RegularXdsKubernetesTestCase):
 
+    # As of 7/6/22 Custom load balancer configuration via xDS is only supported by
```
Comment (sergiitk) [inline tools/run_tests/xds_k8s_test_driver/tests/custom_lb_test.py:37]:
```suggestion
    # As of 2022-07-06 Custom load balancer configuration via xDS is only supported by
```

nit :)

**Comment by temawi** [comment]
Skipping seems to work:

```
❯ ./run.sh tests/custom_lb_test.py --testing_version=v1.41.x
Running tests under Python 3.10.2: /Users/tmwilson/repos/grpc/tools/run_tests/xds_k8s_test_driver/venv/bin/python
I0706 11:57:37.815007 4502353408 xds_k8s_testcase.py:112] ----- Testing CustomLbTest -----
I0706 11:57:37.815078 4502353408 xds_k8s_testcase.py:113] Logs timezone: PDT
I0706 11:57:37.815361 4502353408 skips.py:99] Skipping TestConfig(client_lang='java', server_lang='java', version='v1.41.x')
[  SKIPPED ] setUpClass (__main__.CustomLbTest)
----------------------------------------------------------------------
Ran 0 tests in 0.001s

OK (skipped=1)
```

**Pull request merged**